src/go/rdctl/cmd/root.go

```diff
@@ -41,6 +41,9 @@ var (
   password string
 )
 
+const clientVersion = "1.0.0"
```

Comment:
I'd match this to the app version (i.e. `finalBuildVersion` in `build.mjs`), but we can do that later.

Reply:
Sure.